js/skin.js

```diff
@@ -340,268 +384,338 @@ const Skin = createReactClass({
+                  {...this.props}
+                  closedCaptionOptions={this.props.closedCaptionOptions}
+                  dataItemsPerPage={{ xs: 1, sm: 4, md: 8, lg: 8 }}
+                  responsiveView={this.state.responsiveId}
+                  componentWidth={this.state.componentWidth}
+                />
+              </ContentScreen>
+            );
+            break;
+          case CONSTANTS.SCREEN.VIDEO_QUALITY_SCREEN:
+            screen = (
+              <ContentScreen
+                {...this.props}
+                screenClassName="oo-menu-content-screen"
+                screen={CONSTANTS.SCREEN.VIDEO_QUALITY_SCREEN}
+                titleText={CONSTANTS.SKIN_TEXT.VIDEO_QUALITY}
+                autoFocus={this.state.videoQualityOptions.autoFocus}>
+                <VideoQualityPanel
+                  {...this.props}
+                  fullscreen={this.state.fullscreen}
+                  videoQualityOptions={this.state.videoQualityOptions}
+                  responsiveView={this.state.responsiveId} />
+              </ContentScreen>
+            );
+            break;
+          case CONSTANTS.SCREEN.PLAYBACK_SPEED_SCREEN:
+            screen = (
+              <ContentScreen
+                {...this.props}
+                screenClassName="oo-menu-content-screen"
+                screen={CONSTANTS.SCREEN.PLAYBACK_SPEED_SCREEN}
+                titleText={CONSTANTS.SKIN_TEXT.PLAYBACK_SPEED}
+                autoFocus={this.state.playbackSpeedOptions.autoFocus} >
+                <PlaybackSpeedPanel
+                  language={this.props.language}
+                  localizableStrings={this.props.localizableStrings}
+                  controller={this.props.controller}
+                  skinConfig={this.props.skinConfig}
+                  fullscreen={this.state.fullscreen}
+                  responsiveView={this.state.responsiveId} />
+              </ContentScreen>
+            );
+            break;
+          case CONSTANTS.SCREEN.MULTI_AUDIO_SCREEN:
+            screen = (
+              <ContentScreen
+                {...this.props}
+                cssClass="oo-close-button oo-close-button--ma"
+                dataItemsPerPage={{ xs: 1, sm: 4, md: 8, lg: 8 }}
+                screen={CONSTANTS.SCREEN.MULTI_AUDIO_SCREEN}
+                screenClassName="oo-cc-ma-container"
+                autoFocus={this.state.multiAudioOptions.autoFocus}
+                icon="cc">
+                <ClosedCaptionMultiAudioMenu {...this.props} />
+              </ContentScreen>
+            );
+            break;
+          case CONSTANTS.SCREEN.ERROR_SCREEN:
+            screen = <ErrorScreen {...this.props} errorCode={this.props.controller.state.errorCode} />;
+            break;
+          default:
+            screen = <div />;
+        }
       }
     }
 
+    var className = this.state.responsiveClass;
+    if (this.props.controller.state.audioOnly) {
+      className += " oo-audio-only";
```

Comment:
Can we use the `classNames` utility here?

Reply:
Yep